### ISSUE TITLE
SSH connection fails - Kali - Python3.8

### ISSUE THREAD

**Issue by overjt** [calebstewart/pwncat#27]
Hello I just installed the tool using pip and when I run the command 

`pwncat -s -u SSH_USER -p SSH_PORT -H SSH_HOST -P SSH_PASSWORD`

I get the following error.

```
Traceback (most recent call last):
  File "/home/kali/.local/bin/pwncat", line 8, in <module>                                                                                                                                                                                 
    sys.exit(main())                                                                                                                                                                                                                       
  File "/home/kali/.local/lib/python3.8/site-packages/pwncat/__main__.py", line 28, in main                                                                                                                                                
    pwncat.victim.command_parser.dispatch_line(                                                                                                                                                                                            
  File "/home/kali/.local/lib/python3.8/site-packages/pwncat/commands/__init__.py", line 290, in dispatch_line                                                                                                                             
    command.run(args)                                                                                                                                                                                                                      
  File "/home/kali/.local/lib/python3.8/site-packages/pwncat/commands/connect.py", line 229, in run
    pwncat.victim.connect(chan)
  File "/home/kali/.local/lib/python3.8/site-packages/pwncat/remote/victim.py", line 279, in connect
    hostname_path = self.run("which hostname").strip().decode("utf-8")
  File "/home/kali/.local/lib/python3.8/site-packages/pwncat/remote/victim.py", line 1032, in run
    sdelim, edelim = self.process(cmd, delim=wait, timeout=timeout)
  File "/home/kali/.local/lib/python3.8/site-packages/pwncat/remote/victim.py", line 1086, in process
    data = self.client.recv(len(command), socket.MSG_PEEK)
```

### Local Info
**Operative System:** Kali Linux 2020.2
**Python version:** Python 3.8.3
**User**: non-root


### Remote Info
**Operative System:** Ubuntu 14.04.5 LTS
**User**: non-root

**Comment by calebstewart** [comment]
I recommend installing in a virtual environment. In order to support using SSH directly, I had to implement some features in Paramiko (specifically, the extra parameter to `recv` to align with standard sockets). Those changes have not been merged to the upstream, and therefore pwncat currently requires my fork of paramiko to function w/ SSH. The `setup.py` file references my fork properly but if you already had the mainline version installed, pip would have ignored it.

**Comment by taj0023** [comment]
even after getting an ssh connection, i cant type anything......but i can ctrl+D and type in pwncat term

**Comment by Lameh4ck3r** [comment]
when i type ctrl+ d it suddenly dead 
![image](https://user-images.githubusercontent.com/35825039/88571764-95ab8700-d057-11ea-8855-30d39af44550.png)

**Comment by calebstewart** [comment]
Apologies for the long wait on this. Are you still seeing this error? I cannot reproduce it on my end. It appears to be getting bad results for the `id` command on the target system.

**Comment by pitust** [comment]
I had this issue. You need to uninstall and then reinstall paramiko from the fork.

**Comment by Lameh4ck3r** [comment]
Now its working fine

**Comment by calebstewart** [comment]
Great, I'm going to close this issue, then. If you see it pop up again for whatever reason, please open another issue.
